README.md

```diff
@@ -2,55 +2,28 @@
-privateKey to the dApp. For obvious reasons, outside of testing environments, this is a huge security issue, as the dApp
-could simply use it maliciously stealing funds or signing something not approved by the user.
-
-From a security perspective, providing your key to an application is a **HUGE** risk.  WalletConnect mitigates this risk by delegating all transaction signing operations to
-the user's wallet. Applications instead, given the ability to communicate directly with the wallet to **request** a user's signature.
-
-[WalletConnect](https://walletconnect.org/) is an established solution to tackle this issue, and it's already used by
-many dApps and users.
-
-## Usage example video
+## Example video
 ![Video](demo.gif)
 
-This example is showing [Crypsydra dApp](https://github.com/east-side-cryps/crypsydra-web) connecting with
-[Aero Wallet](https://aero.coz.io)
+This example is showing the [wc-dapp-vite-vanilla](examples/wc-dapp-vite-vanilla) example connecting with
+[Neon Wallet](https://neonwallet.com/) to make a GAS transfer.
+
+## Installation and Setup
+- [WcSdk React](packages/wallet-connect-sdk-react/README.md) - For React.js with streamlined state changes.
+- [WcSdk Core](packages/wallet-connect-sdk-core/README.md) - For other technologies with the same features.
 
-## How does it work?
-- The dApp shows a QRCode, so the user can use their wallet to establish a connection
-    - It's an encrypted websocket connection between dApp and wallet, passing by a relay server
-- Now, every time the dApp needs to call a SmartContract, instead of doing it by itself, it will ask the wallet to do so
-    - It will send a JSON-RPC message with all the information it needs to send to the SmartContract
-    - The wallet will ask for the user to approve this message
-    - It will sign the transaction, make the call and respond back to the dApp with the response from the blockchain
-  
-## Installation and Setup Guide
-This repository contains multiple packages and from here on you need to choose your path. If you are planning to use it
-on a React.js application it's highly recommended using our [WC React SDK](packages/wallet-connect-sdk-react/README.md)
-that handles all state changes for you, but if you plan to use with a different technology we have all the same features
-on the [WC Core SDK](packages/wallet-connect-sdk-core/README.md).
+## Usage
+Check this [Usage Guide](USAGE_GUIDE.md) to see how to use this SDK on your application.
 
-## How to test my dApp?
+## How to test?
 This SDK uses 
 [WalletConnect 2.0](https://docs.walletconnect.org/v/2.0/) protocol and was designed to work on the NEO 3 blockchain.
-You will need a Wallet application that fits these requirements, we recommend
-[Aero Web Wallet](https://melanke.github.io/aero-beta/) for testing or [Neon Wallet](https://neonwallet.com/) for production.
+You will need a Wallet application that fits these requirements, we recommend [Neon Wallet](https://neonwallet.com/).
```

Comment:
The `Wallet Connect Registry` section is missing, it was removed from the SDKs README, but it's not here on the main README.

Reply:
Yea, I did this on purpose, I reviewed their website and couldn't find where to register, so I think we should omit this.